vivisect/__init__.py

```diff
@@ -1073,12 +1072,12 @@ def isProbablyCode(self, va, rerun=False):
         if ret:
             return ret
 
+        rerun = kwargs.pop('rerun', False)
         if va in self.iscode and not rerun:
             return self.iscode[va]
 
         self.iscode[va] = True
-        emu = self.getEmulator()
-        emu.setMeta('silent', True)
```

Comment:
where does this go?  getting rid of 'silent' meta will log a lot more errors while attempting to emulate things that aren't code.

Reply:
So to move our backchannel a bit up to here:

I personally dislike having hidden parameters like that, and slapping another logging configuration on top of the logging module, that already provides the ability to add your own custom logging levels, feels like the wrong thing to do.

So I'll add a custom lower logging level that sits underneath DEBUG.

Reply:
that works. :+1: